doc/conf.py

```diff
@@ -22,7 +22,7 @@
 # -- Project information -----------------------------------------------------
 
 project = 'Ubuntu Install Guide'
-copyright = f'Canonical Group Ltd, {datetime.date.today().year}'
+copyright = f'Canonical Group Ltd, 2020 - {datetime.date.today().year}'
```

Comment:
https://hynek.me/til/copyright-years/   ?? can we drop copyright years altogether now that git history is reasonable enough substrate to read and defend copyright?